src/ripple/app/tx/impl/Escrow.cpp

```diff
@@ -58,22 +58,22 @@ namespace ripple {
     expiration time, the XRP can only be returned to the sender.
 
     For more details on escrow, including examples, diagrams and more please
-    visit https://ripple.com/build/escrow/#escrow
+    visit https://xrpl.org/escrow.html#escrow
```

Comment:
This link will be more stable without the anchor.

```suggestion
    visit https://xrpl.org/escrow.html
```